Source/WebCore/page/Quirks.cpp

```diff
@@ -1620,6 +1620,12 @@ bool Quirks::shouldFlipScreenDimensions() const
 #endif
 }
 
+bool Quirks::shouldAllowDownloadsInSpiteOfCSP() const
+{
+    // FIXME: Remove this when rdar://137625935 is resolved.
+    return isDomain("apple.com"_s);
```

Comment:
Could the scope of the quirk be reduced? `apple.com` is very wide. It would be possible to pass the URL and check the path and/or the full domain name.

Reply:
ok after discussing with @achristensen07 let's hope this is **very temporary**.